packages/ag-grid-theme/css/parts/ag-root-var.css

```diff
@@ -16,7 +16,7 @@ div[class*="ag-theme-salt"] {
   --ag-font-family: var(--salt-text-fontFamily);
   --ag-font-size: var(--salt-text-fontSize);
   --ag-foreground-color: var(--salt-content-primary-foreground);
-  --ag-grid-size: var(--salt-spacing-50);
+  --ag-grid-size: var(--salt-spacing-100);
```

Comment:
TODO: this impacts menu sizes, which is not wanted. This was changed for filter and menu icon spacing on header cells. need a different solution

Reply:
```
.ag-ltr .ag-header-cell:not(.ag-right-aligned-header) .ag-header-label-icon, .ag-ltr .ag-header-cell:not(.ag-right-aligned-header) .ag-header-menu-icon {
    margin-left: var(--ag-grid-size);
}
```